### PR TITLE
Improve jira and artifactory login note

### PR DESCRIPTION
**Pull request by NotMyFault**
According to infra.ci, both jobs run every other hour. I've improved the Jira login notice too.

### CODE REVIEW

File: src/main/groovy/io/jenkins/infra/repository_permissions_updater/ArtifactoryPermissionsUpdater.groovy [hunk 1]
```diff
@@ -240,7 +240,7 @@ class ArtifactoryPermissionsUpdater {
                                 """
                                 ${developer} needs to log in to [Artifactory](https://repo.jenkins-ci.org/) and [Jira](https://issues.jenkins.io/).
 
-                                We resync our Artifactory list hourly, so you will need to wait some time before rebuilding your pull request.
+                                We resync our Artifactory and Jira user list bihourly, so you will need to wait some time before rebuilding your pull request.
```
Comment (timja) [inline src/main/groovy/io/jenkins/infra/repository_permissions_updater/ArtifactoryPermissionsUpdater.groovy:243]:
```suggestion
                                We resync our Artifactory and Jira user list every 2 hours, so you will need to wait some time before rebuilding your pull request.
```

File: src/main/groovy/io/jenkins/infra/repository_permissions_updater/ArtifactoryPermissionsUpdater.groovy [hunk 2]
```diff
@@ -253,7 +253,7 @@ class ArtifactoryPermissionsUpdater {
                                         """
                                 ${developer} needs to log in to [Artifactory](https://repo.jenkins-ci.org/).
 
-                                We resync our Artifactory list hourly, so you will need to wait some time before rebuilding your pull request.
+                                We resync our Artifactory user list bihourly, so you will need to wait some time before rebuilding your pull request.
```
Comment (timja) [inline src/main/groovy/io/jenkins/infra/repository_permissions_updater/ArtifactoryPermissionsUpdater.groovy:256]:
```suggestion
                                We resync our Artifactory user list every 2 hours, so you will need to wait some time before rebuilding your pull request.
```

File: src/main/groovy/io/jenkins/infra/repository_permissions_updater/ArtifactoryPermissionsUpdater.groovy [hunk 3]
```diff
@@ -264,7 +264,12 @@ class ArtifactoryPermissionsUpdater {
                             if (!existsInJira) {
                                 reportChecksApiDetails(developer + " needs to log in to Jira",
                                         """
-                                ${developer} needs to log in to [Jira](https://issues.jenkins.io/).
+                                ${developer} needs to log in to [Jira](https://issues.jenkins.io/)
+
+                                We resync our Jira user list bihourly, so you will need to wait some time before rebuilding your pull request.
```
Comment (timja) [inline src/main/groovy/io/jenkins/infra/repository_permissions_updater/ArtifactoryPermissionsUpdater.groovy:269]:
```suggestion
                                We resync our Jira user list every 2 hours, so you will need to wait some time before rebuilding your pull request.
```

**Review by timja — COMMENTED** [review]
bihourly is not a common word (chrome spellcheck doesn't know about it even) best to simplify I think

**Comment by NotMyFault** [comment]
> bihourly is not a common word (chrome spellcheck doesn't know about it even) best to simplify I think

rip, let's use something simpler then

**Review by timja — APPROVED** [review]

**Pull request merged**